pySDC/tests/test_Runge_Kutta_sweeper.py

```diff
@@ -8,6 +8,10 @@
     'Cash_Karp': 'teal',
 }
 
+EXPLICIT_METHODS = ['RK1', 'MidpointMethod', 'CrankNicholson']
```

Comment:
Is `RK1` Forward Euler or Backward Euler ? And why is `MidpointMethod` both in `EXPLICIT_METHODS` and `IMPLICIT_METHODS` ?

Reply:
I implemented most RK rules following the [Wikipedia page on list of RK methods](https://en.wikipedia.org/wiki/List_of_Runge–Kutta_methods). There is for instance both explicit and implicit Midpoint. 
Maybe it wasn't the best decision, but instead of separate classes, you supply as a parameter which version you want. Same is true for RK1. If you give `RK1` as sweeper class with `sweeper_params = {'implicit': True}`, you get backward Euler and vice versa.

Reply:
Then maybe `ExplicitMidpoint`, `ImplicitMidpoint`, `ForwardEuler` and `BackwardEuler` so the names are more explicit ?